packages/strapi-parts/src/EmptyStateLayout/EmptyStateLayout.stories.mdx

```diff
@@ -33,9 +45,9 @@ Description...
   </Story>
 </Canvas>
 
-## EmptyStateLayout without action
+### EmptyStateLayout without action
 
-Description...
+If no need to redirect the user to a more relavant place or if the context does not suit for that type of action, an EmptyState can be without any particular action. 
```

Comment:
"can be without any particular action" looks there's something missing in the sentance